index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>サイト制作課題</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Anton&family=Noto+Sans+JP:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="header-logo-wrapper">
+        <img class="header-logo" src="./img/logo.svg" alt="Grand Players" />
+      </h1>
+    </header>
+    <section class="hero">
+      <div class="hero-container">
+        <div class="hero-texts">
+          <p class="hero-main-copy">
+            BE<br />
```

Comment:
```suggestion
            BE<br>
```
改行を表す要素は`<br/>`ではなくて`<br>`と書きます。

Reply:
どちらでも同様に改行として解釈されるので、好みの問題が大きいかなと思います。

どちらが適切かについてはいろいろな意見があるようですが、可読性はどうしても主観に寄ってしまうのでそこまで意識しなくても大丈夫です。

参考：https://dev.to/scalertopics/difference-between-and-tag-in-html-5350